module-system/module-implementations/sov-chain-state/src/tests.rs

```diff
@@ -0,0 +1,185 @@
+use sov_data_generators::value_setter_data::ValueSetterMessages;
```

Comment:
Can we make tests to be integration tests? This way we always ensure that public API is exposed correctly.

Reply:
I feel that should be part of its own separate PR to avoid overloading this one